about/faq.rst

```diff
@@ -380,7 +380,7 @@ Godot provides its own type-casting system, which can optionally use RTTI
 internally. Disabling RTTI in Godot means considerably smaller binary sizes can
 be achieved, at a little performance cost.
 
-Why does Godot not force users to implement DoD (Data oriented Design)?
+Why does Godot not force users to implement DoD (Data-Oriented Design)?
```

Comment:
If it's **DoD**, then it should be capitalized **D**ata-**o**riented **D**esign.